build/images/ovs/apply-patches.sh

```diff
@@ -57,6 +57,10 @@ curl https://github.com/openvswitch/ovs/commit/79eadafeb1b47a3871cb792aa972f6e4d
 curl https://github.com/openvswitch/ovs/commit/7cc77b301f80a63cd4893198d82be0eef303f731.patch | \
     git apply
 
+# This patch (post 2.13.0) ensures that ct_nw_src/ct_nw_dst supports IP Mask.
+curl https://github.com/openvswitch/ovs/commit/1740aaf49dad6f533705dc3dce8d955a1840052a.patch | \
```

Comment:
@antoninbas , I try to apply this patch on my local OVS, the new code works well, and I could use a MASK in match ct_nw_src/ct_nw_dst. But if I try torun integration tests in containers, it fails, and the failure looks the patch doesn't work. Could you help update the image antrea/openvswitch to ensure the fix on ct_nw_xxx could be included?

Reply:
@wenyingd I can push an image but it there is a job re-building and updating the image on dockerhub every 12 hours, so it will get overwritten pretty fast.
the best thing to do is probably to wait until this PR has been approved and is ready to be merged. Then either Quan or I can build the new docker image locally and push it to dockerhub, and we can re-trigger the integration tests job.

Reply:
@antoninbas Got it. Then does it mean I need to remove the corresponding integration test from this PR, and add it to another one after this PR is merged? Otherwise, the tests won't pass.

Reply:
That's one possibility. Alternatively, you can run the tests locally, and once the PR is approved and read to be merged, I can push the image and we can run the tests then so CI succeeds.

Is the PR ready for review now? Or are you still working on it?

Reply:
It is ready for review now.

Reply:
@wenyingd I pushed the modified docker image to dockerhub and ran the CI integration tests again. However they are still failing because it turns out that the integration tests install OVS using the Ubuntu package manager instead of using our openvswitch docker image.

you will need to modify this dockerfile so that the base image is `antrea/openvswitch:2.13`: https://github.com/vmware-tanzu/antrea/blob/master/build/images/test/Dockerfile. I think you will need to install go inside the docker, since you will not be able to use `golang:1.13` as the base image anymore.

Finally the image I pushed on dockerhub will be overwritten in 6 hours. If you are not done by then, you can ask @tnqn to push the image again for you. Otherwise, I can do it again tomorrow.